Test/AwsCommonRuntimeKitTests/mqtt/MqttRRClientTests.swift

```diff
@@ -72,11 +88,35 @@ class Mqtt5RRClientTests: XCBaseTestCase {
                 print(contextName + " MqttRRClientTests: onLifecycleEventDisconnection")
                 disconnectionExpectation.fulfill()
             }
+            
+            self.onRRIncomingPublish = { publishEvent in
+                self.callbackRWLock.write {
+                    self.rrPublishEvent.append(publishEvent)
+                    self.incomingPublishExpectation.fulfill()
+                }
+            }
+            
+            self.onSubscriptionStatusUpdate = { [self] statusEvent in
+                self.callbackRWLock.write {
+                    self.subscriptionStatusEvent = statusEvent
+                    print(contextName + " MqttRRClientTests: onSubscriptionStatusUpdate. EventType: \(statusEvent.event)")
+                    if statusEvent.event == SubscriptionStatusEventType.established {
+                        self.subscriptionStatusSuccessExpectation.fulfill()
+                    } else {
+                        if let error = statusEvent.error {
+                            print(contextName + " MqttRRClientTests: onSubscriptionStatusUpdate failed with error : (\(error.code)) \(error.name) : \(error.message)")
+                        }
+                        self.subscriptionStatusErrorExpectation.fulfill()
+                    }
+                }
+            }
+            
         }
         
-        // release the context before exit the test case
-        func cleanup(){
+        // make sure cleanup the resources before exit the test caseto
```

Comment:
Trivial: "make sure to cleanup" or "make sure we cleanup" and there's a trailing 'to' after the end of 'case'